src/main/fc/config.c

```diff
@@ -527,30 +528,43 @@ bool readEEPROM(void)
     return success;
 }
 
-void writeEEPROM(void)
+static void _writeEEPROM(bool setConfigured)
```

Comment:
We're not much using the `_<functionName>` pattern for internal functions so far. should we add this to the coding style?

Reply:
Nah, I've just done the `_<functionName>` for quick hacking. Will try renaming it to something else.